src/SFA.DAS.Admin.Roatp.Web/Controllers/ProviderStatusUpdateController.cs

```diff
@@ -22,8 +22,8 @@ public async Task<IActionResult> Index(int ukprn, CancellationToken cancellation
 
         var model = new OrganisationStatusUpdateViewModel
         {
-            OrganisationStatus = organisationResponse.Status,
-            OrganisationStatuses = BuildOrganisationStatuses(organisationResponse.Status)
+            OrganisationStatus = organisationResponse.Content!.Status,
```

Comment:
create a variable for organisationResponse.Content! and use that in the controller instead of all the references